src/BlazorUI/Bit.BlazorUI/Components/Buttons/ActionButton/BitActionButton.scss

```
@@ -1,72 +1,113 @@
-    padding: var(--bit-acb-padding);
-    font-size: var(--bit-acb-fontsize);
     letter-spacing: $tg-ctrl-letter-spacing;
     text-transform: $tg-ctrl-text-transform;
     -webkit-tap-highlight-color: transparent;
-    transition: color $mot-duration-short $mot-easing;
+    color: var(--bit-ActionButton-color, #{$clr-fg-pri});
+    padding: var(--bit-ActionButton-padding, var(--bit-acb-padding));
+    font-size: var(--bit-ActionButton-font-size, var(--bit-acb-fontsize));
+    border-radius: var(--bit-ActionButton-radius, #{$shp-radius-button});
+    background-color: var(--bit-ActionButton-background, transparent);
+    // The one color the icon and the spinner read; every state below retargets it along with the text color.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)